specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -3080,6 +3080,10 @@
         "noise": {
           "description": "Properties describing noise level of the image.",
           "$ref": "#/definitions/Noise"
+        },
+        "mask": {
```

Comment:
mask attributes should be mentioned in "detect" description

Reply:
Added.